src/js/components/DataChart/stories/StackedBars.js

```diff
@@ -18,27 +18,30 @@ const Example = () => (
     <Box align="center" justify="start" pad="large">
       <DataChart
         data={data}
+        property={[
```

Comment:
This example no longer shows Stacked Bars
![image](https://user-images.githubusercontent.com/6320236/87095252-a194ed80-c1fd-11ea-85ee-93221ac82b68.png)

Reply:
oops, fixed.